db/migrate/20211203194815_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer the above similar comment for the other migration. :pray: